From b154d44282e41c7dc9779d490eb9a8c3c5a9f4fe Mon Sep 17 00:00:00 2001
From: Shammah Chancellor <shammah.chancellor@gmail.com>
Date: Mon, 27 Jun 2022 19:34:05 -0700
Subject: [PATCH 169/196] Bump version number to 3.3.3 (#261)

* Fixes address bug which caused the testnet to become corrupted
  (Did not effect mainnet)
* Drop testnet difficulty to 1
* Use same address sets for testnet, mainnet, and regtest to avoid
  future confusion.
* Fix off by one error in nng interface for chronik and other services
---
 CMakeLists.txt             | 2 +-
 contrib/aur/lotus/PKGBUILD | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 20f1f6c83..e89f49454 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -7,7 +7,7 @@ set(CMAKE_USER_MAKE_RULES_OVERRIDE
 )
 
 project(lotus
-	VERSION 3.2.3
+	VERSION 3.3.3
 	DESCRIPTION "Lotus is a full node implementation of the Lotus protocol."
 	HOMEPAGE_URL "https://www.givelotus.org"
 )
diff --git a/contrib/aur/lotus/PKGBUILD b/contrib/aur/lotus/PKGBUILD
index e13533330..476d9a2c5 100644
--- a/contrib/aur/lotus/PKGBUILD
+++ b/contrib/aur/lotus/PKGBUILD
@@ -1,7 +1,7 @@
 # Maintainer: Josh Ellithorpe <quest@mac.com>
 
 pkgname=lotusd
-pkgver=3.2.3
+pkgver=3.3.3
 pkgrel=0
 pkgdesc="Lotus with lotusd, lotus-tx, lotus-seeder and lotus-cli"
 arch=('i686' 'x86_64')
-- 
2.39.5 (Apple Git-154)

